nextcord/channel.py

```diff
@@ -578,6 +584,8 @@ async def create_webhook(self, *, name: str, avatar: Optional[bytes] = None, rea
         from .webhook import Webhook
 
         if avatar is not None:
```

Comment:
This can be changed to simply `if avatar:`, just a suggestion, up to you depending on the scope of the PR.

Reply:
I don't think changing this is necessary. I would say checking that it is not None is better since you are explicitly checking if the parameter was set to something, rather than checking if it's truthy. In this case, it might not make a difference in practice, but theoretically, it could make a difference (eg. there could be some falsy bytes-like object the user might want to use as their default).